### DISCUSSION TITLE
Releases

### DISCUSSION THREAD

**Discussion by andreykosteev** [microsoft/azurelinux#6663] [General] [answered]
Hello! Could you tell me, do you have a release schedule? And when is the next release with vulnerability fixes planned?

**Comment by mfrw** [comment, marked as answer]
Hello @andreykosteev;
Yes, we typically release twice every-month. The first release is during the start of the month, while as the second one is towards the end of the month.

Please take a look at our releases page for more information: https://github.com/microsoft/CBL-Mariner/releases

**Reply by andreykosteev** [reply]
Yes, I'm monitoring new releases on the "Releases" page. Many thanks for the information about the release schedule.
